product-service/serverless.yml

```diff
@@ -6,6 +6,14 @@ provider:
   stage: dev
   region: eu-west-1
 
+  # dbConfig.json with private data was not added to repo
+  environment:
```

Comment:
[proposal]
It's great that db settings is not commited to the repository, but I still propose to use .env file because it's more common than json